obstacles_x: [ 25.0, 35.0, 35.0  ]
obstacles_y: [ 0.0, 2.0, -2.0 ]
obstacles_r: [1.0, 2.0, 0.5]